fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/mixin/content/registry/MixinAbstractFurnaceBlockEntity.java

```diff
@@ -34,4 +36,19 @@
 	private static void fuelTimeMapHook(CallbackInfoReturnable<Map<Item, Integer>> info) {
 		FuelRegistryImpl.INSTANCE.apply(info.getReturnValue());
 	}
+
+	@Inject(at = @At("HEAD"), method = "canUseAsFuel", cancellable = true)
+	private static void canUseAsFuelHook(ItemStack stack, CallbackInfoReturnable<Boolean> info) {
```

Comment:
Both new injects are soft overwrites, any other mod injecting here will never be handled.

Reply:
Would it be better to inject just before `createFuelTimeMap` ? That should work too.

Reply:
But if I were to do that, any mod injecting at TAIL for example still wouldn't work I think. Is it really reasonable to expect mods to be injecting stuff in these two functions?

Reply:
If you want I can lower the priority on the mixin itself... that works too, doesn't it?

Reply:
Resolved by d0ce38b.